recipe/gen_patch_json.py

```diff
@@ -987,6 +987,14 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             constrains.append(f"cudatoolkit {cuda_major_minor}")
             record['constrains'] = constrains
 
+        if record_name == "ucx" and record.get('timestamp', 0) < 1682924400000:
+            constrains = record.get('constrains', [])
+            for i, c in enumerate(constrains):
+                if c.startswith('cudatoolkit'):
+                    v = c.split()[-1]
+                    constrains[i] = c = f'cudatoolkit {c}|{c}.*'
```

Comment:
```suggestion
                    constrains[i] = f'cudatoolkit {c}|{v}.*'
```

Reply:
Yeah spotted the same thing when testing locally. Wanted to confirm the change first before pushing